### ISSUE TITLE
151

### ISSUE THREAD

**Issue by johnnieng** [johnnieng/DP-100#77]
QUESTION 151
You create a multi-class image classification deep learning model that uses the PyTorch deep learning framework.
You must configure Azure Machine Learning Hyperdrive to optimize the hyperparameters for the classification model.
You need to define a primary metric to determine the hyperparameter values that result in the model with the best
accuracy score.
Which three actions must you perform? Each correct answer presents part of the solution.
NOTE: Each correct selection is worth one point.
A. Set the primary_metric_goal of the estimator used to run the bird_classifier_train.py script to
maximize.
B. Add code to the bird_classifier_train.py script to calculate the validation loss of the model and log
it as a float value with the key loss.
C. Set the primary_metric_goal of the estimator used to run the bird_classifier_train.py script to
minimize.
D. Set the primary_metric_name of the estimator used to run the bird_classifier_train.py script to
accuracy.
E. Set the primary_metric_name of the estimator used to run the bird_classifier_train.py script to
loss.
F. Add code to the bird_classifier_train.py script to calculate the validation accuracy of the model
and log it as a float value with the key accuracy.
